jupyterlab/tests/test_app.py

```diff
@@ -122,5 +129,80 @@ def _process_finished(self, future):
             os._exit(1)
 
 
+class KarmaTestApp(TestApp):
+    """A notebook app that runs the jupyterlab karma tests.
+    """
+    karma_pattern = Unicode('src/*.spec.ts')
+    karma_base_dir = Unicode('')
+
+    def get_command(self):
+        """Get the command to run."""
+        terminalsAvailable = self.web_app.settings['terminals_available']
```

Comment:
does this setting still make sense now that terminals work on Windows?

Reply:
Support for terminals depends on the not-yet-released Notebook 5.3

Reply:
Ahh, got it!